examples/HILO_Symbol_Plotting.py

```diff
@@ -0,0 +1,158 @@
+                clip_on=True, horizontalalignment='center', verticalalignment='center',
+                transform=transform)
+        ax.text(lon[mxy[i], mxx[i]], lat[mxy[i], mxx[i]], '\n'+str(np.int(data[mxy[i], mxx[i]])),
+                color=color, size=12, clip_on=True, fontweight='bold',
+                horizontalalignment='center', verticalalignment='top', transform=transform)
+
+
+###############################
+# Get NARR data
+dattim = datetime(1999, 1, 3, 0)
+
+ncss = NCSS('http://nomads.ncdc.noaa.gov/thredds/ncss/grid/narr-a/{0:%Y%m}/{0:%Y%m%d}/'
+            'narr-a_221_{0:%Y%m%d}_{0:%H}00_000.grb'.format(dattim))
+query = ncss.query()
+query.all_times().variables('Pressure_reduced_to_MSL',
+                            'Geopotential_height').add_lonlat().accept('netcdf4')
+data = ncss.get_data(query)
+
+###############################
+# Extract data into variables
+
+# Grab pressure levels
+plev = list(data.variables['isobaric'][:])
+
+# Grab lat/lons and make all lons 0-360
+lats = data.variables['lat'][:]
+lons = data.variables['lon'][:]
+lons[lons < 0] = 360 + lons[lons < 0]
+
+# Grab valid time and get into datetime format
+time = data['time']
+vtime = num2date(time[:], units=time.units)
+
+# Grab MSLP and smooth, use MetPy Units module for conversion
+EMSL = data.variables['Pressure_reduced_to_MSL'][:] * units.Pa
+EMSL.ito('hPa')
+mslp = gaussian_filter(EMSL[0], sigma=3.0)
+
+# Grab pressure level data
+hght_1000 = data.variables['Geopotential_height'][0, plev.index(1000)]
+hght_500 = data.variables['Geopotential_height'][0, plev.index(500)]
+
+# Calculate and smooth 1000-500 hPa thickness
+thickness_1000_500 = gaussian_filter(hght_500 - hght_1000, sigma=3.0)
+
+###############################
+# Set map and data projections for use in mapping
+
+# Set projection of map display
+mapproj = ccrs.LambertConformal(central_latitude=45., central_longitude=-100.)
+
+# Set projection of data
+dataproj = ccrs.PlateCarree()
+
+# Grab data for plotting state boundaries
+states_provinces = cfeature.NaturalEarthFeature(
+        category='cultural',
+        name='admin_1_states_provinces_lakes',
+        scale='50m',
+        facecolor='none')
+
+###############################
+# Create figure and plot data
+fig = plt.figure(1, figsize=(17., 11.))
+ax = plt.subplot(111, projection=mapproj)
+
+# Set extent and plot map lines
+ax.set_extent([-145., -70, 20., 60.], ccrs.PlateCarree())
+ax.coastlines('50m', edgecolor='black', linewidth=0.75)
+ax.add_feature(states_provinces, edgecolor='black', linewidth=0.5)
+
+# Plot thickness with multiple colors
+clevs = (np.arange(0, 5400, 60), 5400, np.arange(5460, 7000, 60))
+colors = ('tab:blue', 'b', 'tab:red')
+for clevthick, color in zip(clevs, colors):
+    cs = ax.contour(lons, lats, thickness_1000_500, levels=clevthick, colors=color,
+                    linewidths=1.0, linestyles='dashed', transform=dataproj)
+    plt.clabel(cs, fontsize=11, inline=True, inline_spacing=5, fmt='%i',
```

Comment:
Since these `clabel` kwargs are all the same for both clabels, we should probably make them a dictionary that is used in both.

Reply:
done